packages/gp2-components/src/templates/OutputForm.tsx

```diff
@@ -461,6 +478,7 @@ const OutputForm: React.FC<OutputFormProps> = ({
             )}
 
             <div css={containerStyles}>
+              {createVersion && <OutputVersions versions={versions} />}
```

Comment:
I think we need to pass `versionAction` to have the correct style and copy. The left side is figma and right side the hub

![Screenshot 2023-12-18 at 11 51 48](https://github.com/yldio/asap-hub/assets/16595804/63d02df1-534d-47a3-9735-b29d86fa75ba)